src/common/pf_dcp.c

```diff
@@ -1301,9 +1304,15 @@ static int pf_dcp_identify_req (
    uint8_t block_error;
    pnet_ethaddr_t mac_address;
 
+   /* For diagnostic logging */
+   bool identify_all = false;
+   uint16_t stationname_position = 0;
+   uint16_t stationname_len = 0;
+   uint16_t alias_position = 0;
```

Comment:
These variables will be unused if logging is disabled. Some analysis tools (such as clang-tidy) will probably consider this an error.

Reply:
Done